pkgs/top-level/all-packages.nix

```diff
@@ -14051,6 +14051,8 @@ in
 
   lwan = callPackage ../servers/http/lwan { };
 
+  fasttext = callPackage ../applications/science/machine-learning/fasttext { };
```

Comment:
It should rather be in the `### APPLICATIONS` section, right after `exrtools`
(and yes `labelImg` is also misplaced...)

Reply:
Thanks! Fixed.